README.md

```diff
@@ -38,11 +38,11 @@ At the moment, the following contract files use the experimental ABIEncoderV2 So
 - `pm/MixinTicketBrokerCore.sol`
 - `pm/MixinWrappers.sol`
 
-There have been bugs related to ABIEncoderV2 in the past and it is still experimental so developers should pay attention to the [list of bugs associated with ABIEncoderV2](https://solidity.readthedocs.io/en/latest/bugs.html) when making any contract code changes that involve ABIEncoderV2 and should make sure to use a compiler version with the necessary fixes. The primary motivation behind enabling ABIEncoderV2 in these contract files is to allow for Solidity structs to be passed as function arguments. 
+There have been bugs related to ABIEncoderV2 in the past and it is still experimental so developers should pay attention to the [list of bugs associated with ABIEncoderV2](https://solidity.readthedocs.io/en/latest/bugs.html) when making any contract code changes that involve ABIEncoderV2 and should make sure to use a compiler version with the necessary fixes. The primary motivation behind enabling ABIEncoderV2 in these contract files is to allow for Solidity structs to be passed as function arguments.
 
-### Install 
+### Install
 
-Make sure Node.js v10.17.0 is installed.
+Make sure Node.js v14.17.4 is installed.
```

Comment:
Since this is LTS this should also work with prior versions I think. We can remove this line